.ci_support/win_64_.yaml

```diff
@@ -0,0 +1,14 @@
+c_compiler:
+- vs2019
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+libpng:
+- '1.6'
+openjpeg:
+- '2'
+target_platform:
+- win-64
+zlib:
+- '1.2'
```

Comment:
Can we add support for AEC compression?  Looks like libaec is available in conda

https://anaconda.org/conda-forge/libaec

Reply:
This file is created by "conda smithy rerender" and I don't know why libaec isn't included here, but others are.  

libaec is listed as a host requirement for all platforms in the recipe/meta.yaml.

My hypothesis is that libaec isn't listed in .ci_support/win_64_.yaml because I put minimum version constraint on libaec in recipe/meta.yaml, but not the other packages.

Reply:
I think you need to list `libaec` once without a constraint and once with a constraint in `host` for conda-smithy to pick it up and put it in the `.ci_support` files as it should.  I've just added this.